assets/prototype/domain/shared/predicates/priceTypes/selectionPredicates/index.ts

```diff
@@ -24,6 +24,8 @@ export const isNotTax = propEq('isTax', false);
 
 // returns true if supplied price type is a flat fee (dollar) surcharge
 export const isFlatFeeSurcharge = allPass([isNotBasePrice, isNotDiscount, isNotPercent]);
+// returns array of price types that are NOT base price types
+export const getPriceModifiers = (priceTypes: PriceType[]): PriceType[] => filter(isNotBasePrice, priceTypes);
```

Comment:
this same function is in the selection predicates file for Prices, could maybe build an abstracted one that they could share, but it might start to get messy